### ISSUE TITLE
table(Platform): On accessing the cells within the table as highlighted, column, row count information is not conveyed by the screen reader.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3841]
Description: On accessing the cells within the table as highlighted, column, row count information is not conveyed by the screen reader.

Expected: Count information shall be conveyed along with the UI information within the table.

Screenshot: 
[Column header.docx](https://github.com/SAP/fundamental-ngx/files/6220800/Column.header.docx)

![image](https://user-images.githubusercontent.com/52071505/122205378-cdb0f800-cebd-11eb-82e8-e822e3797fde.png)

**Comment by sKudum** [comment]
Updating title as per the image attached.

**Comment by dimamarksman** [comment]
@puru-hk the image resolution is really low, I can't see what is on the picture
The simillar problem for others issues

**Comment by puru-hk** [comment]
@Sanjay-1312 Could you please check this issue. If it is not fixed. Could you please attach the proper screenshot.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

I can still able to reproduce the issue.
On accessing the 1st  row the second row details is being read and the count is not read.
So please fix the issue.

Thanks and Regards
Sanjay

**Comment by manjunathanagaraj** [comment]
@puru-hk , Lets follow the process we have. I think the issue should have had label 'QA approved'. If this is not working, QA should ideally route is back to developer. Also, lets mention about the platform - browser and OS combination.

**Comment by platon-rov** [comment]
VoiceOver on Mac pronounces rows & columns count, please look. It pronounces counts one time, on entering the table, then - only content of the cell. If leave & re-enter the table - it will pronounce rows & columns count again.

![image](https://user-images.githubusercontent.com/20265336/113275041-0261da00-92e7-11eb-932c-88bc6cc95295.png)

And about:

> On accessing the 1st row the second row details is being read and the count is not read.

For me VoiceOver read the text right (see screenshot above). So, could you please provide more details for reproducing the issue? Maybe some of the bugs are reproducible only with certain screen reader?

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is not fixed.
On accessing the column first row additional information is conveyed.

Expected: Count information shall be conveyed along with the UI information within the table.
URL:https://fundamental-ngx.netlify.app/#/platform/table
JAWS 2021

Screenshot:
![image](https://user-images.githubusercontent.com/78714518/119930979-21ef4900-bf9e-11eb-8ec8-d4a9683c3394.png)

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is not fixed.
OB: On accessing the table first row, the row and column details is read additional information such as row 1 and colum1 is read and on accessing the table cell data information is not read.
![image](https://user-images.githubusercontent.com/78714518/124061843-403cde80-da4d-11eb-9aa1-fe43e51fff4e.png)
EB: ideally, the table cell details should be read and only relevant column and row details should be read and the data cell information.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

OB: Additional information  such Column and row 1 is read and the  row 2 is being that is the wrong information when column  2 of row 1 is accessed and loading additional information
![image](https://user-images.githubusercontent.com/78714518/124240715-a3f00600-db38-11eb-9e62-a0184bc04f23.png)
EB: Additional information should be removed such as the row and column of  1 and Loading information should be removed and  correct column and row information such as column2 and row 1 should be read and when ever the table row and column accessed should be read precisely throughout the table.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

OB: On accessing the column row of second and remaining rows the count is not read.
EB: Ideally, the row and column count should be read to the user.
![image](https://user-images.githubusercontent.com/78714518/125911262-ca825f8d-c695-44c9-a9c4-1b5a4f9ddfd1.png)

JAWS 2021
https://deploy-preview-5692--fundamental-ngx.netlify.app/#/platform/table

**Comment by platon-rov** [comment]
@Sanjay-1312 here is mine:

![image](https://user-images.githubusercontent.com/20265336/125925093-dc4dfaee-115c-4bfe-b8a0-b754f2059a63.png)

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

I can still able to reproduce the issue.
I have added screenshot for your reference,
Screenshot:
![image](https://user-images.githubusercontent.com/78714518/126111572-3a1ce6c7-9bd4-44d0-875a-da8bb06e6479.png)

**Comment by platon-rov** [comment]
@Sanjay-1312 here is mine

![image](https://user-images.githubusercontent.com/20265336/126274136-b7f2f491-9930-49b9-8915-9cdb17c36340.png)

Which browser/OS do you have?

**Comment by puru-hk** [comment]
@platon-rov Accessibility team used Chrome browser and windows 10 OS.

**Comment by Veerakarthick** [comment]
Hello Colleague,

Retest has been taken below are the details.

issue1:
Description: On accessing the cells within the table as highlighted, column, row count information is not conveyed by the screen reader.

Expected: Count information shall be conveyed along with the UI information within the table.
screenshot:
![image](https://user-images.githubusercontent.com/37998745/151785772-4211ff07-2e62-477b-b966-185f07159d4f.png)

Retest Environment:
URL:https://deploy-preview-5692--fundamental-ngx.netlify.app
browser:Chrome Version 96.0.4664.110 (Official Build) (64-bit)
Screen Reader: Jaws2021.2111.13 ILM

Regards,
Karthikeyan.c

**Comment by Karthickc1994** [comment]
Hello colleague,

Retest has been taken results are below:

Issue1: Still reproducible:

Observed behavior:
When accessing the table data cell the corresponding the table column header association and row and column count information is not read out.

Screen shot:
![image](https://user-images.githubusercontent.com/105037808/167574636-23e94ee5-38ba-419b-9b27-63803f7645aa.png)


Expected behavior:
when accessing the table data cell the column and row count information should read out.
